firefox-ios/Client/metrics.yaml

```diff
@@ -656,6 +656,67 @@ downloads:
       - fx-ios-data-stewards@mozilla.com
     expires: "2025-01-01"
 
+# Microsurvey
+microsurvey:
+  survey_message_shown:
+    type: event
+    description: |
+        The survey surface (bottom sheet) was shown and visible.
+    bugs:
+      -
+    data_reviews:
+      -
+    notification_emails:
+      - fx-ios-data-stewards@mozilla.com
+    expires: "2025-01-01"
+
+  privacy_notice_tapped:
+    type: event
+    description: |
+        Records that the user tapped on the privacy notice.
+    bugs:
+      -
+    data_reviews:
+      -
+    notification_emails:
+      - fx-ios-data-stewards@mozilla.com
+    data_sensitivity:
+      - interaction
+    expires: "2025-01-01"
+
+  dismiss_button_tapped:
+    type: event
+    description: |
+        Records that the user tapped on the close button to dismiss the survey.
+    bugs:
+      -
+    data_reviews:
+      -
+    notification_emails:
+      - fx-ios-data-stewards@mozilla.com
+    data_sensitivity:
+      - interaction
+    expires: "2025-01-01"
+
+  primary_survey_button_tapped:
```

Comment:
Maybe `submit_button_tapped`? What do you think?

Reply:
works for me!